acinclude.m4

```diff
@@ -1483,7 +1483,6 @@ AC_DEFUN([CURL_PREPARE_BUILDINFO], [
     curl_pflags="${curl_pflags} CYGWIN"
   fi
   case $host_os in
-    msys*) curl_pflags="${curl_pflags} MSYS";;
```

Comment:
Where exactly is host_os set? Don't we want to keep this for backward compatibility with older toolchains? Or, probably alter the line to say CYGWIN instead of MSYS.

Reply:
autotools sets it automatically. This change only affects the displayed platform flags in `buildinfo.txt`; `MSYS` will no longer appear there (but that's only informational really). `CYGWIN` still continue to appear if the host string contains either `cygwin` or `msys` (for backward compatibility):
https://github.com/curl/curl/blob/d1fc1c4a854d5cc809cc4ae59a9511900228023a/configure.ac#L604-L607